### PR TITLE
[enhance] optional regex flags

### PR DESCRIPTION
**Pull request by fabianlee1211**
Hi there Farhad,

Tried to add an optional regex flags field for other fellow devs to have more control on the regex pattern.
Hopefully it's a good idea.

Cheers!

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -4,12 +4,17 @@ import * as github from '@actions/github';
 async function run() {
   try {
     const titleRegex = core.getInput('title-regex', {required: true});
+    const regexFlags = core.getInput('regex-flags');
```
Comment (seferov) [inline src/main.ts]:
```suggestion
    const regexFlags = core.getInput('title-regex-flags');
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ jobs:
     - uses: seferov/pr-lint-action@master
       with:
         title-regex: '^\[PROJECT-\d*\]\ '
+        regex-flags: 'g'
 ```
 
-In this example, for every pull request the title is expected to match `^\[PROJECT-\d*\]\ ` regex. For instance, `[PROJECT-123] lorem ipsum` or `[PROJECT-2345] dolor sit amet` are valid titles for this example. You can customize the title regex for your needs.
+In this example, for every pull request the title is expected to match `^\[PROJECT-\d*\]\ ` regex with a global flag `g`. For instance, `[PROJECT-123] lorem ipsum` or `[PROJECT-2345] dolor sit amet` are valid titles for this example. You can customize the title regex for your needs. The regex flags is optional.
```
Comment (seferov) [inline README.md]:
```suggestion
In this example, for every pull request the title is expected to match `^\[PROJECT-\d*\]\ ` regex with a global flag `g`. For instance, `[PROJECT-123] lorem ipsum` or `[PROJECT-2345] dolor sit amet` are valid titles for this example. You can customize the title regex for your needs. The regex flags configuration is optional.
```

**Comment by fabianlee1211** [comment]
Updated based on your suggestions!
Thanks for accepting!

**Comment by seferov** [comment]
Thank you @fabianlee1211

**Pull request merged**